src/access.rs

```diff
@@ -68,9 +68,7 @@ where
 			.lock()
 			.unwrap()
 			.sync(&self.blockchain, sync_options)
-			.map_err(|e| Error::Bdk(e))?;
-
-		Ok(())
+			.map_err(|e| Error::Bdk(e))?
```

Comment:
Oh, I think, drop the `?`, too.

Reply:
The error handling will be cleaned up in #11, in particular we'll stop exposing upstream errors to users.